src/plugins/intel_gpu/src/plugin/ops/rnn.cpp

```diff
@@ -140,7 +138,7 @@ static void CreateLSTMCellOp(Program& p, const std::shared_ptr<ngraph::op::v4::L
     cldnn::tensor FCInputReshapeSz = { lstm_batch_size, inputShape.spatial[0] + inStateShape.spatial[0], 1, 1 };
     p.add_primitive(*op, cldnn::reshape(FCInputReshapeID, cldnn::input_info(input_concatID), FCInputReshapeSz));
 
-    p.add_primitive(*op, cldnn::fully_connected(lstm_fc_id, cldnn::input_info(FCInputReshapeID), WRconcatID, hasBias ? bias.pid : ""));
+    p.add_primitive(*op, cldnn::fully_connected(lstm_fc_id, cldnn::input_info(FCInputReshapeID), WRconcatID, bias.pid));
```

Comment:
is this also coverity? This is quite a huge PR and if it doesn't belong to coverity fix, I think it would be better to put it in a separate PR.

Reply:
Yes, immutable `hasBias` value leads to dead code